<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppMainWindow</class>
 <widget class="QMainWindow" name="AppMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="qpb_load">
        <property name="text">
         <string>Load plugins...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qpb_unload">
        <property name="text">
         <string>Unload plugins</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="hb::plugin::HbPluginListWidget" name="plw_plugins" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="qmb_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
    <addaction name="qa_load_unload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="qa_load_unload">
   <property name="text">
    <string>Load / Unload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>hb::plugin::HbPluginListWidget</class>
   <extends>QWidget</extends>
   <header>HbPluginListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
